.husky/commit-msg

```diff
@@ -1,11 +1,4 @@
-#!/usr/bin/env sh
```

Comment:
according to the [Husky 9.0.1 release](https://github.com/typicode/husky/releases/tag/v9.0.1) we can remove this line